share/nrnunits.lib.in

```diff
@@ -63,19 +63,15 @@ demi-			.5
 / constants
 
 fuzz			1
+two			2
 pi			3.14159265358979323846
 c			2.99792458+8 m/sec fuzz
 g			9.80665 m/sec2
 au			1.49597871+11 m fuzz
-mole			6.022169+23 fuzz
-mol             1
```

Comment:
I think that by removing `mol` there is going to be an issue with some mod files as the comment underneath suggests